util/image_viewer/spotfinder_frame.py

```diff
@@ -96,8 +96,51 @@ def __init__(self, *args, **kwds):
 
         super().__init__(*args, **kwds)
 
+        self.viewing_stills = True
+        for explst in self.experiments:
+            for exp in explst:
+                if exp.scan is not None or exp.goniometer is not None:
+                    self.viewing_stills = False
+                    break
```

Comment:
I don't think we have false-y scans or goniometers so possibly this simplifies to
```suggestion
        for experiment_list in self.experiments:
            if any(e.scan or e.goniometer for exp in experiment_list):
                self.viewing_stills = False
                break
```
which would also fix the `break` logic (currently this only breaks out of the current iteration of the inner loop)